client/blocks/signup-form/index.jsx

```diff
@@ -1100,6 +1102,19 @@ class SignupForm extends Component {
 			: formState.getFieldValue( this.state.form, 'email' );
 	};
 
+	getPasswordlessInviteHeaderText = () => {
+		const { invite } = this.props;
+		if ( invite?.site.title ) {
+			return this.props.translate( 'Sign up to start editing {{span}}%(domain)s{{/span}}', {
```

Comment:
A site should always have a title, but I'm not 100% sure.
What about `invite?.site?.title` ?
And we should maybe offer an alternate text if we don't have a title.

Reply:
Site title is always present event if empty on wp-admin, in that case it uses the site URL as site title.
It breaks because the site URL has no spaces. To prevent this layout issue, I will add a CSS word-break.